src/foraging_gui/RigJsonBuilder.py

```diff
@@ -274,7 +273,6 @@ def build_rig_json_core(settings, water_calibration, laser_calibration):
             solenoid_valve=d.Device(
                 device_type="Solenoid", 
                 name="Solenoid Left",
-                manufacturer=d.Organization.LEE,
```

Comment:
Did this company get removed from the schema models? That seems odd

Reply:
I couldn't find it [here](https://github.com/AllenNeuralDynamics/aind-data-schema-models/blob/main/src/aind_data_schema_models/organizations.py) so I removed it. If you can find it though I can add it back in.

Reply:
It looks like its now defined in this file, as "The Lee Company"
https://github.com/AllenNeuralDynamics/aind-data-schema-models/blob/main/src/aind_data_schema_models/models/organizations.csv

Reply:
Oh okay i wasn't aware of the csv model. I will figure it out and add back in